exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/xsort/managed/PriorityQueueCopierWrapper.java

```diff
@@ -82,7 +84,7 @@ private PriorityQueueCopier newCopier(VectorAccessible batch) {
     ClassGenerator<PriorityQueueCopier> g = cg.getRoot();
     cg.plainJavaCapable(true);
     // Uncomment out this line to debug the generated code.
-//    cg.saveCodeForDebugging(true);
+    cg.saveCodeForDebugging(true);
```

Comment:
I guess the fiddle-factor is getting to high; time to implement your suggestion of having options to enable this stuff. I'll try to add that in some future PR.